completor/prepare_outputs.py

```diff
@@ -91,42 +91,121 @@ def dataframe_tostring(
+                        ("CONNECTION", "FACTOR"): "{:.10g}".format,
+                        ("FORMATION", "PERMEABILITY", "THICKNESS"): "{:.10g}".format,
+                        ("MEASURED", "DEPTH"): "{:.3f}".format,
+                        ("TRUE", "VERTICAL", "DEPTH"): "{:.3f}".format,
+                        ("START", "MEASURED", "DEPTH"): "{:.3f}".format,
+                        ("START", "MD"): "{:.3f}".format,
+                        ("END", "MEASURED", "DEPTH"): "{:.3f}".format,
+                        ("END", "MD"): "{:.3f}".format,
+                        ("FLOW", "COEFFICIENT"): "{:.10g}".format,
+                        ("FLOW", "CROSS", "SECTIONAL", "AREA"): "{:.3e}".format,
+                        ("OIL", "FLOW", "CROSS", "SECTIONAL", "AREA"): "{:.3e}".format,
+                        ("GAS", "FLOW", "CROSS", "SECTIONAL", "AREA"): "{:.3e}".format,
+                        ("WATER", "FLOW", "CROSS", "SECTIONAL", "AREA"): "{:.3e}".format,
+                        ("MAX", "FLOW", "CROSS", "SECTIONAL", "AREA"): "{:.3e}".format,
+                        ("WATER", "HOLDUP", "FRACTION", "LOW", "CUTOFF"): "{:.10g}".format,
+                        ("WATER", "HOLDUP", "FRACTION", "HIGH", "CUTOFF"): "{:.10g}".format,
+                        ("GAS", "HOLDUP", "FRACTION", "LOW", "CUTOFF"): "{:.10g}".format,
+                        ("GAS", "HOLDUP", "FRACTION", "HIGH", "CUTOFF"): "{:.10g}".format,
+                        ("ALPHA", "MAIN"): "{:.10g}".format,
+                        ("ALPHA", "PILOT"): "{:.10g}".format,
+                    }
+                )
+                if column_splits[0][0].startswith("--"):
+                    # Make sure each level is commented out!
+                    column_splits[0] = tuple(["--"] * number_of_levels)
+                # Replace nan with empty for printing purposes.
+                new_cols = pd.DataFrame(column_splits).fillna("")
+                df_temp.columns = pd.MultiIndex.from_frame(new_cols)
+
+    try:
+        output_string = df_temp.to_string(
+            index=False, justify="justify", formatters=formatters, header=header, sparsify=False
+        )
+    except ValueError:
+        if df_temp.isnull().values.any():
+            raise CompletorError("Got NaN values in table, please report if encountered!")
+        df_temp.replace("*", "1*", inplace=False)
+        columns_with_1_star = df_temp.columns[df_temp.eq("1*").any()]
+        df_temp = df_temp.replace("1*", np.nan, inplace=False)
+        # Probably find columns where this is the case and cast to numeric after replacing with nan?
+        df_temp[columns_with_1_star] = df_temp[columns_with_1_star].astype(np.float64, errors="ignore")
+        output_string = df_temp.to_string(
+            index=False, justify="justify", formatters=formatters, header=header, sparsify=False, na_rep="1*"
+        )
+
     if output_string is None:
         return ""
+
+    limit = 132
```

Comment:
This should be max 128, maybe even lower just to be safe?

Reply:
Maxed it at 128!